pyalgotrading/algobulls/connection.py

```diff
@@ -183,9 +185,15 @@ def delete_previous_trades(self, strategy):
         Args:
             strategy: Strategy code
         """
-        response = self.api.delete_previous_trades(strategy)
-        if response['data'] != 'success':
-            print(f'Deletion of previous trade data for strategy "{strategy}" may not have been successful. Details: {response}')
+        response = {}
+        for _ in range(30):
+            response = self.api.delete_previous_trades(strategy)
+            if response.get('data') == 'success':
+                print(response.get('message'))
+                break
+            else:
+                print(f'deleting previous trades ... attempt [{_}])\n{response}\n')
```

Comment:
Capitalize the first word; There should be no space before `...`

Reply:
done